main.js

```diff
@@ -0,0 +1,28 @@
+const hamburger = document.querySelector('.ham-btn1');
+const overlay = document.querySelector('.overlay');
+const menu = document.querySelector('.overlaymenu1');
+const cancel = document.querySelector('.cancel');
+const port = document.querySelector('.items-port');
+const contact = document.querySelector('.items-contact');
+const about = document.querySelector('.items-about');
+
+hamburger.addEventListener('click', () => {
+  menu.style.display = 'block';
+  overlay.style.display = 'block';
+});
+
+cancel.addEventListener('click', () => {
+  overlay.style.display = 'none';
+});
+
+port.addEventListener('click', () => {
+  overlay.style.display = 'none';
+});
+
+about.addEventListener('click', () => {
+  overlay.style.display = 'none';
+});
+
+contact.addEventListener('click', () => {
+  overlay.style.display = 'none';
+});
```

Comment:
- [ ] Great working on this menu so far but I think the design is not customized on the portfolio width since the close button is so far so please customzise it on the same width 

- Your design
![image](https://user-images.githubusercontent.com/58553711/189913376-d0daf990-5b6c-4ae5-a8b4-c33f33c5c6e2.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/189913868-d5b5bae5-dc6f-4dac-a767-2fe91dfdc006.png)